ui/docker/40-envsubst-on-reearth_config.sh

```
@@ -0,0 +1,42 @@
+
+    if [ ! -f "$template_file" ]; then
+        entrypoint_log "$ME: ERROR: template file $template_file does not exist"
+        return 1
+    fi
+
+    if [ ! -w "$(dirname "$output_file")" ]; then
+        entrypoint_log "$ME: ERROR: $(dirname "$output_file") is not writable"
+        return 1
+    fi
+
+    defined_envs=$(printf '${%s} ' $(env | cut -d= -f1 | awk "/${filter}/ {print}"))
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)